lighthouse-core/module/index.js

```diff
@@ -18,9 +18,9 @@
 'use strict';
 
 const semver = require('semver');
-const log = require('../src/lib/log.js');
-const ChromeProtocol = require('../src/lib/drivers/cri.js');
-const lighthouse = require('../src/lighthouse');
+const Runner = require('../runner');
```

Comment:
Yeah, so I've sat there and looked through these files myself a bunch of times for this exact reason!

* module/index is the require('lighthouse') file so you can use it in Node, and it uses the runner
* the extension also uses the runner
* runner is the shared handler (for both module & extension) that needs to be given a driver, and sets up the core to do its job
* core does the gathering, auditing and aggregation

```
module/index \
              ----- Runner ----- Core [Gather / Audit / Aggregate] 
extension    /